packages/components/src/components/PipelineRun/PipelineRun.jsx

```diff
@@ -112,16 +122,6 @@ export /* istanbul ignore next */ class PipelineRunContainer extends Component {
     }));
   };
 
-  getPipelineTask = ({ pipelineRun, selectedTaskId, taskRun }) => {
```

Comment:
Class method expected to use `this`. No need for it to be a class method since it's a pure function so move it outside the component and update references